django/cantusdb_project/main_app/forms.py

```diff
@@ -67,6 +66,16 @@ def label_from_instance(self, obj):
         return obj.name
 
 
+class SelectWidgetNameModelChoiceField(NameModelChoiceField):
+    """
+    This class inherits from NameModelChoiceField, but uses the
+    the custom SelectWidget defined in widgets.py as its widget
+    (for styling).
+    """
+
+    widget = SelectWidget()
+
+
 # 3 best
```

Comment:
```suggestion

```
I don't know why this comment is here I saw it earlier when I was working on a different issue. Maybe we can just remove it.

Reply:
Yeah, I was wondering the same thing....I think if neither of us want to keep it, I'll remove!